crates/bevy_window/src/window.rs

```diff
@@ -145,6 +145,40 @@ impl WindowResizeConstraints {
     }
 }
 
+/// Mirror of `Icon` from `winit`.
+#[derive(Debug, Clone)]
+pub struct WindowIcon {
+    /// 32-bit-per-pixel RGBA image data.
```

Comment:
I am not keen on exposing something so raw, but I'm not sure it's worth the complexity to fix now.